pages/events/[event].tsx

```diff
@@ -121,44 +133,84 @@ const EventPage: NextPage = () => {
         // This redirect won't work on localhost (can't add dynamic URLs to
         // Supabase's allow-list), but does work in prod
         redirect={
-          eventID
-            ? `${process.env.NEXT_PUBLIC_HOSTNAME || ""}/events/${eventID}`
-            : undefined
+          eventID ? `${process.env.NEXT_PUBLIC_HOSTNAME || ""}/events/${eventID}` : undefined
         }
       />
     );
   }
-
+  let startDateStr = "",
+    endDateStr = "";
+  let startDate = new Date();
```

Comment:
Prefer not using `let` if we can help it — another way to do this might be something like:
```
if (!eventData || dateStatus === DateStatus.loading) {
    return "Loading...";
}
// now we know eventData !== null && dateStatus !== DateStatus.loading
const startDate = new Date(eventData.start_date)
startDateStr = ...